plugins/csp-trajectories/src/Plugin.hpp

```diff
@@ -34,7 +34,8 @@ class Plugin : public cs::core::PluginBase {
 
         /// Without this, some versions of clang claim this type to be non-default-constructible...
         /// Also, Trail() = default; does not work...
-        Trail(){};
+        Trail() = default;
+        ;
```

Comment:
Either the comment or the code is wrong now.

Reply:
This was "auto-fixed" by clang-tidy. Now it should work again... MSVC still complains. I'll see about that on Monday.